megamek/src/megamek/client/ui/swing/dialog/AbstractUnitSelectorDialog.java

```diff
@@ -854,16 +862,10 @@ public Class<?> getColumnClass(int col) {
             return getValueAt(0, col).getClass();
         }
 
-        @Override
-        public boolean isCellEditable(int row, int col) {
```

Comment:
I assume this is false in the base class?

Reply:
Yes. (IDEA told me this is equal to the base class)